ec2/sc-ec2-linux-jumpcloud-workflows-v1.0.0.yaml

```diff
@@ -429,6 +429,10 @@ Resources:
           Value: "yes"
         - Key: "PatchGroup"
           Value: "prod-default"
+          #The session name appended to this roleid becomes the value of the AccessApprovedCaller tag
+        - Key: "AccessApprovedRoleId"
```

Comment:
why not add this to the other EC2 templates as well?

Reply:
same question -- if we know this works, we need edits on the other product templates

Reply:
There's no harm in adding it to all the templates as long as access methods are confusing to endusers. Maybe people will want to shell into their notebooks. Let's put the tag on Windows instances too and at some point we can change the boot scripts to apply the Protected/AccessApprovedCaller tag if we think it would be useful.